commit 1c84f1f980ed36e95e0bc410a7955c569bf4b4d2
Author: Steve Dickson <steved@redhat.com>
Date:   Thu May 10 15:04:07 2007 -0400

    The wrong bit field is being passed to NFSCTL_TCPISSET()
    during one of the sanity checks in rpc.nfsd.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/nfsd/nfsd.c b/utils/nfsd/nfsd.c
index d0bbfb3..aaf8d29 100644
--- a/utils/nfsd/nfsd.c
+++ b/utils/nfsd/nfsd.c
@@ -118,7 +118,8 @@ main(int argc, char **argv)
 		fprintf(stderr, "no version specified\n");
 		exit(1);
 	}			
-	if (NFSCTL_VERISSET(versbits, 4) && !NFSCTL_TCPISSET(versbits)) {
+
+	if (NFSCTL_VERISSET(versbits, 4) && !NFSCTL_TCPISSET(protobits)) {
 		fprintf(stderr, "version 4 requires the TCP protocol\n");
 		exit(1);
 	}
